tools/ml-dataprep/runner.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Bootstrap module in charge of launching the extraction of the ML datasets
+from BigQuery."""
+
+
+import argparse
+import dataprep
+import metadata
+
+
+def initialise_parameters(args_parser):
+    """Define the arguments with the default values, parses the arguments
+    passed to the main program, and set the PARAMETERS global variable.
+
+    Args:     args_parser
+    """
+
+    # Data files arguments
+    args_parser.add_argument(
+        '--parameters',
+        help='Columns parameters that should be replaced in the names.',
+        required=False
+    )
+    args_parser.add_argument(
+        '--source_project',
+        help='GCP project from where to extract the data.',
+        required=True
+    )
+    args_parser.add_argument(
+        '--source_dataset',
+        help='BigQuery dataset from where to extract the data.',
+        required=True
+    )
+    args_parser.add_argument(
+        '--source_table',
+        help='BigQuery table from where to extract the data.',
+        required=True
+    )
+    args_parser.add_argument(
+        '--destination_project',
+        help='GCP project where to save the intermediary data.',
+        required=True
+    )
+    args_parser.add_argument(
+        '--destination_dataset',
+        help='BigQuery dataset where to save the intermediary data.',
+        required=True
+    )
+    args_parser.add_argument(
+        '--destination_gcs_path',
+        help='Google Cloud Storage where to export the training and validation data in CSV format.',
+        required=True
+    )
+    args_parser.add_argument(
+        '--split_ratio',
+        help='Split ratio for the train/validation split.',
+        required=False,
+        default=0.75,
+        type=float
+    )
+    return args_parser.parse_args()
+
+
+def get_column_parameters():
+    """Transform to array a list input parameter."""
+    return [] if PARAMETERS.parameters is None else PARAMETERS.parameters.split(',')
+
+
+def main():
+    """Create training and validation datasets."""
+    print('Parameters:')
```

Comment:
Logging instead of print (see previous comment).

Reply:
Done in 20b5bb97a6147d7fc178e42d542643c5b81a08c5